src/api/procedures/moveFunds.ts

```diff
@@ -26,6 +38,61 @@ export type Params = MoveFundsParams & {
   from: DefaultPortfolio | NumberedPortfolio;
 };
 
+/**
+ * @hidden
+ * separates user input into fungible and nft movements
+ */
+async function segregateItems(
+  items: PortfolioMovement[],
+  context: Context
+): Promise<{
+  fungibleMovements: PortfolioMovementFungible[];
+  nftMovements: PortfolioMovementNonFungible[];
+}> {
+  const fungibleMovements: PortfolioMovementFungible[] = [];
+  const nftMovements: PortfolioMovementNonFungible[] = [];
+  const tickers: string[] = [];
+
+  for (const item of items) {
```

Comment:
can we pull out `const { asset } = item;` here once so that we don't have to always add in `item.asset` ?